extension/content.css

```diff
@@ -213,26 +213,21 @@
 	color: #df3e3e;
 }
 
-/* hide news items when people pushed to a branch */
-.news .alert.push {
+/* hide news items when people:
+pushed to a branch
+create a branch
+deletes a branch
+adds someone as a collaborator
+forks a repo
```

Comment:
LOL, don't mean to argue, but they should all be past tense. They happened already, so we got a notification. :wink:

Also going to add `!important` a la https://github.com/sindresorhus/refined-github/pull/22#discussion_r53374626